src/bp-forums/groups.php

```diff
@@ -320,6 +369,53 @@ public function edit_screen( $group = false ) {
 			}
 		}
 
+		/**
+		 * Exclude the forum if it is associated with other groups.
+		 * Exclude the forum if the forum type is category.
+		 * Exclude the forum if the forum is child forum.
+		 *
+		 * @since BuddyBoss 1.7.6
+		 *
+		 * @param array $forum_id  Fourm ids.
+		 * @param int   $group_id  Group id.
+		 *
+		 * @uses bbp_get_forum() Get forum.
+		 *
+		 * @return bool
+		 */
+		public function forum_can_associate_with_group( $forum_id, $group_id ) {
+
```

Comment:
move this function to the end of the file.

Reply:
Move the function at the end of the file.
https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/b2c663fb81f0630d02690c0ac5064d62ff9f8387